e2e/test/scenarios/dashboard-filters/reproductions/16663-filters-can-stay-in-url.cy.spec.js

```diff
@@ -0,0 +1,65 @@
+import {
+  restore,
+  editDashboard,
+  saveDashboard,
+  visitDashboard,
+} from "e2e/support/helpers";
+
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
+
+const { ORDERS_ID } = SAMPLE_DATABASE;
+
+const questionDetails = {
+  query: {
+    "source-table": ORDERS_ID,
+  },
+};
+
+const parameters = [
+  {
+    name: "Quarter and Year",
+    slug: "quarter_and_year",
+    id: "f8ae0c97",
+    type: "date/quarter-year",
+    sectionId: "date",
+    default: "Q1-2023",
+  },
+];
+
+const dashboardDetails = { parameters };
+
+describe("issue 16663", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("should remove filter value from url after going to another dashboard (metabase#16663)", () => {
+    const dahsboardToRedirect = "Orders in a dashboard";
+
+    cy.createQuestionAndDashboard({ questionDetails, dashboardDetails }).then(
+      ({ body: dashboardCard }) => {
+        const { dashboard_id } = dashboardCard;
+
+        visitDashboard(dashboard_id);
+      },
+    );
+
+    editDashboard();
+
+    cy.get("main header").icon("gear").click();
+
+    saveDashboard();
```

Comment:
```suggestion
```

These 3 actions are probably unnecessary now given we map the parameter default value since the dashboard creation.

Could you try removing them and see if the test still works?

Reply:
indeed, with default parameter those are not needed, thanks